content/pt-br/_index.html

```diff
@@ -0,0 +1,113 @@
+---
+title: Istio
+description: Connect, secure, control, and observe services.
+---
+<!-- these script blocks are only for the primary English home page -->
```

Comment:
Please remove this script block. As the comment says, this should only be on the top-level English home page.